homeassistant/components/zha/fan.py

```diff
@@ -262,8 +263,6 @@ async def async_added_to_hass(self) -> None:
 class IkeaFan(BaseFan, ZhaEntity):
     """Representation of a ZHA fan."""
 
-    _attr_name: str = "IKEA fan"
```

Comment:
Why did this remove a key?

Reply:
It's a base fan, no point in calling it an IKEA fan.